frontend/src/components/dashboard/common/PeriodDropdown.vue

```diff
@@ -1,38 +1,38 @@
 <template>
-    <VSelect
+    <DefaultDropdown
         :items="items"
-        item-text="title"
-        item-value="value"
-        class="option-select no-details"
         :value="value"
-        solo
-        return-object
         @change="changePeriod"
     />
 </template>
 
 <script>
     import {period} from "@/services/periodService";
+    import DefaultDropdown from '@/components/common/DefaultDropdown';
 
     export default {
         name: "PeriodDropdown",
+        components: {
+            DefaultDropdown
+        },
         props: {
             value: {
                 type:String,
                 required: true
             },
-        },
-        data() {
-            return {
-                items: [
-                    {title: 'Today', value: period.PERIOD_TODAY},
-                    {title: 'Yesterday', value: period.PERIOD_YESTERDAY},
-                    {title: 'Last 7 days', value: period.PERIOD_LAST_WEEK},
-                    {title: 'Last month', value: period.PERIOD_LAST_MONTH},
-                    {title: 'Last quartal', value: period.PERIOD_LAST_QUARTAL},
-                    {title: 'All period', value: period.PERIOD_ALL}
-                ]
-            };
+            items: {
```

Comment:
Not quite sure if you need to define items as a prop here, it's better to do this in local state, or in computed prop, if you want to make it immutable for sure (not a blocker)